torchaudio/_internal/misc_ops.py

```diff
@@ -0,0 +1,30 @@
+from typing import Union, Callable
```

Comment:
nit: That's a very generic name for the file :)

Reply:
Yeah, I do not know what to call, but this is just evicting these functions from `__init__.py` so if this module glows into something then we can have a better naming.